compose.yml

```
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+services:
+  site:
+    build: "."
+    image: "ozone-site-dev"
+    ports:
+      - "3000:3000"
+    volumes:
+      - ".:/ozone-site"
+      - /ozone-site/node_modules/
+    working_dir: "/ozone-site"
+    command: sh -c "pnpm start --host '0.0.0.0'"
```

```suggestion
    command: "pnpm start --host '0.0.0.0'"
```

For some reason this takes ~10 seconds for docker compose to stop if `sh` is used, but it stops instantly if `pnpm` is invoked directly. Something with the sigint/sigterm handling I think.

---

Thanks for suggesting the improvement.
Solved in the latest commit

---

Thanks for suggesting the improvement.
Solved in the latest commit